app/src/main/java/com/framgia/englishconversation/utils/binding/BindingUtils.java

```diff
@@ -3,6 +3,7 @@
 import android.databinding.BindingAdapter;
 import android.graphics.drawable.Drawable;
 import android.net.Uri;
+import android.support.annotation.NonNull;
```

Comment:
ERROR: Unused import: android.support.annotation.NonNull. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports